packages/pymedphys_dicom/src/pymedphys_dicom/dicom/dose.py

```diff
@@ -227,8 +271,8 @@ def find_dose_within_structure(structure_name, dcm_struct, dcm_dose):
 
 
 def create_dvh(structure, dcm_struct, dcm_dose):
-    structure_dose_values = find_dose_within_structure(
-        structure, dcm_struct, dcm_dose)
+    structure_dose_values = find_dose_within_structure(structure, dcm_struct,
+                                                       dcm_dose)
```

Comment:
Swapped to using a different auto formatter ...

Reply:
I actually prefer this if line length permits it :)